### ISSUE TITLE
Shared Mobility Operations update - Mary Vo.

### ISSUE THREAD

**Issue by jaime-mckeown** [cityofaustin/atd-data-tech#3485]
Mary updated the data for the Shared Mobility Operations feature layer for AGOL and asked me to re-publish. She also asked if I could help with projecting the micro-mobility provider data into state plane.

**Comment by jaime-mckeown** [comment]
- Projected the micro-mobility provider data to state plane.
- Noticed that she had Groups in her Mxd, emailed her back that Groups are not currently allowed in AGOL and asked if she wanted to update the map before I published it.

**Comment by jaime-mckeown** [comment]
Mary updated the map to remove the Groups and modified the data. She added a new provider that needed to be projected and had a question about how best to show the data since it covers multiple areas instead of one like the others.

**Comment by jaime-mckeown** [comment]
- Update Revel polygon data to be a multi-part polygon and added new field for the total number of permits.
- There were a couple errors that needed to be fixed before the feature layer could be published. I needed to change a couple of the symbols, kept getting advanced rendering errors. Finally able to publish.
- Asked Mary if they had any graphics for Shared Mobility to use for the thumbnails in AGOL. She provided a wonderful super colorful graphic that I added to the feature layer, Web Map, and Web App.
